web/frontend/src/components/slam-2d-render.vue

```diff
@@ -295,20 +223,22 @@ const updatePointCloud = (pointcloud: Uint8Array) => {
   const viewWidth = viewHeight * 2;
 
   const points = loader.parse(pointcloud.buffer);
+  pointsMaterial = points.material as THREE.PointsMaterial;
+  pointsMaterial.sizeAttenuation = false;
+  pointsMaterial.size = 4;
```

Comment:
can this please be in a constant at the top of the file?

Reply:
Added!